### DISCUSSION TITLE
Link Leaderboard to multiple Games

### DISCUSSION THREAD

**Discussion by SporyTike** [RetroAchievements/RAWeb#778] [Ideas]
I would like to have the possibility to link a leaderboard to multiple game entries. As an example every pokemon game has an entry, but games like Pokemon Leafgreen and Firered have the same Trainer Tower which tracks the time you needed for beating all trainers. It doesn't really make sense to have a leaderboard for each game if there is no difference so it would be good to just submit all entries into the same leaderboard so to be the master of the leaderboard you only need one good run.
Same situation for New Super Mario Bros. and Super Mario 64 DS. There are 17 minigames which are exactly the same and it would be good to just allow to link the leaderboards to both games instead of having different leaderboards for exactly the same thing.

**Comment by Prism7** [comment]
This would also be great for bonus sets. I imagine Super Mario 64 DS will have the exact same leaderboards copied over from the main set.
